### ISSUE TITLE
download_models CLI command creates additional directories when executed in non-root directory

### ISSUE THREAD

**Issue by vs74** [neuropoly/axondeepseg#394]
**Describe the bug**
When we run `download_models` cli command  in a directory different from root directory of the project, it creates additional directories (`AxonDeep/models/`) and store the models (**default_SEM_model** and **default_TEM_model**) in them.

**Expected Behaviour**
What we expect is that when this command is executed in termial, we should have the models stored in the following path `axondeepseg/AxonDeepSeg/models/default_SEM_model` or `axondeepseg/AxonDeepSeg/models/default_TEM_model`.

**To Reproduce**
1. Go to `AxonDeepSeg` directory 
2. In the terminal, run `download_models` cli command

You'll find the models stored not in the `AxonDeepSeg/models` directory, but what you'll see is that  models would be stored in `AxonDeepSeg/AxonDeepSeg/models/default_SEM_model`and `AxonDeepSeg/models/default_TEM_model`

**Desktop :**
 - OS: MacOS Mojave

**Comment by mathieuboudreau** [comment]
Is this being missed by a current test?

**Comment by vs74** [comment]
Yes, it is. The main reason for this behaviour is that we are using harcoded paths in the [download_model.py ](https://github.com/neuropoly/axondeepseg/blob/master/AxonDeepSeg/models/download_model.py)

https://github.com/neuropoly/axondeepseg/blob/e2cd6fcc7502db6241cbea4b7b47e1201599315d/AxonDeepSeg/models/download_model.py#L8-L9

**Solution:**
There is quick solution to it by replacing these paths by the following code 
```   
     sem_destination = Path(__file__).parent / "default_SEM_model"
        tem_destination = Path(__file__).parent / "default_TEM_model"
```

**Comment by vs74** [comment]
I can open a PR to resolve this issue or can resolve it in this [PR](https://github.com/neuropoly/axondeepseg/pull/387).

**Comment by mathieuboudreau** [comment]
Resolved in #717
